packages/cloud_functions/cloud_functions/test/firebase_functions_test.dart

```diff
@@ -125,6 +126,32 @@ void main() {
           FirebaseFunctions.instance.useFunctionsEmulator(origin: '');
         }, throwsA(isA<AssertionError>()));
       });
+
+      test('handles "localhost" and "127.0.0.1" origin only for Android', () {
+        const testLocalhostOrigins = [
+          'http://127.0.0.1:5000',
+          'http://localhost:5000',
+        ];
+        const platforms = TargetPlatform.values;
+
+        platforms.forEach((platform) {
```

Comment:
prefer `for (... in)` instead

Reply:
@rrousselGit `for (var ...)` or `for (TargetPlatform ...)`?

Reply:
```dart
for (final platform in TargetPlatform.values) {
 ...
}
```

And same thing for `testLocalhostOrigins`